---
title: "BI410 Final Project: Lit Review and Workflow"
author: "Mindy Tran"
date: "11/15/2021"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

##  Literature review

This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.

When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:

# 1. Introduce the problem and explain why

Drug overdose represents the leading cause of accidental death in the United States (McLaughlin). Specifically, opioids are a highly addictive narcotic that are prescribed for pain management but, when abused, can lead to dependence and overdose (Chetna). There are numerous factors that impact an individual’s risk for dependence, and genetic studies of opioid use suggest that drug use is heritable (Chetna). There is specific interest in the habenula as it has been linked to neuropsychiatric disorders like addiction and depression (Velasquez, McLaughlin, Li, Darcq). Drug withdrawal syndromes and aversive states like anxiety and depression share similar physiological manifestations in the habenula. The signaling is linked to reward pathways that include the habenula (McLaughlin), thus making the habenula a target to understand the effects of drugs of abuse on the brain (Velasquez). Signaling from the lateral habenula inhibits dopaminergic neurons, which can help explain the negative effects of cocaine use. Here, I hope to use data from the Sylwestrak Lab to see how chronic morphine use impacts gene expression in the habenula.


# 2. Past work and data available on the project

In a 2015 paper, Mocosko et al. published a paper on a new method for single-cell RNA sequencing: Droplet-based single cell sequencing. The method was developed to determine the gene expression on individual cells in a sample. For my data, this process is not done in-lab but it is good to know where our data is coming from and how we get our data. The authors identified 44,808 mouse retinal cells and further analysis identified 39 transcriptionally distinct cell populations in the retina. With this method of sequencing, the authors created an atlas of gene expression for known and novel candidate retinal subtypes for their dataset (Mocosko, 2015). This workflow to identify cell populations can be applied to my mice habenula data.

A 2020 paper by Wallace et al. also looked at mice habenula. Overall, they used single-cell RNA sequencing data to infer functionality of different cell subtypes. They also found gene markers for different anatomical regions of the habenula. I used the markers from this paper to isolate the neurons in my data. For example, Tac2 and corresponds to the MHb, Gap43 and corresponds to the LHb (Wallace, 2020).

(Pandey, 2018)
A paper from Pandey et al. used single single-cell RNA sequencing data in 2018 to discover thirteen new neuronal types in the habenula. They also identified fine-grained spatial subdivisions and discovered dozens of new marker genes that define habenular neuronal types. I used the markers from this paper to isolate the habenula cells in my data.

More recently, a paper from 2020 identified cell-type markers for different brain cell types and habenula cell subtypes. The data the authors have comes from both zebrafish and mice. I used the markers from this paper to isolate the neurons in my data (Hashikawa, 2020). The markers include:
-	Opalin and Mog marked oligodendrocyte clusters
-	Slc4a4 and Ntsr2 marked astrocyte clusters
-	Pdgfra, Gpr17, or Ccnd1 marked oligodendrocyte precursor cells (OPCs)
-	Tmem119 and C1qc marked microglia
-	Tagln marked the mural cell cluster
-	Cldn5 and Flt1 marked the endothelial cells
-	Fam216b marked ependymal cluster

Darq et al. studied RSK2 in 2012, and their data suggests that RSK2 may contribute to nociception, or the perception of pain. The study also suggests there is a signaling pathway between the mu opioid receptor and RSK2. The study indicates that the pathway contributes to acute morphine analgesia, or the inability to feel pain from acute morphine use (Darcq, 2012).


A study from Li et all. showed that the βCaMKII (a kinase) may be powerful regulator of LHb neuron function and a key molecular determinant of depression. The data showed that βCaMKII is up-regulated in the LHb of animal models of depression (Li, 2013). I hope to also study this kinase and how it is regulated in an animal model of addiction.

The Mageswary et al. study mapped of morphine-induced localization of the mu opioid receptor gene, oprm1, in zebrafish brain. Since there are various areas with oprm1 expression, the study implies morphine has multiple action targets. Also, the study suggests oprm1+ cells may have brain functions including reward, and negative emotions (Mageswary, 2020). I also hope to look at the mu opioid receptor in my data to see if the localization in zebrafish translates to our mice model.




# 3. Purpose of the study
The data I will combine and use is droplet-based single-cell RNA sequencing data. I will compare the gene expression of the control experiment with saline injections, and experiment with morphine injections. The animal model is mice. Also, I want to identify what I think are the best cell type markers for the clusters in my labs data. With our data, I want to see if our gene markers differ from the current literature. Possibly, I hope to add markers to the list of accepted markers for cell subtypes in the habenula. 


# 4. Hypotheses/questions:
The questions I have include: What is the affect of chronic morphine use on gene expression in the opioid signaling pathway? Which cell types show enriched expression of opioid related genes? 
I predict that, since βCaMKII is up-regulated in the LHb of animal models of depression (Li, 2013), and many studies mention overlap between depression and addiction, βCaMKII will also be up-regulated in our animal model of addiction.


# 5. Literatured cited

Darcq, E., Befort, K., Koebel, P. et al. RSK2 Signaling in Medial Habenula Contributes to Acute Morphine Analgesia. Neuropsychopharmacol 37, 1288–1296 (2012). https://doi.org/10.1038/npp.2011.316

Hashikawa, Yoshiko, et al. “Transcriptional and Spatial Resolution of Cell Types in the Mammalian Habenula.” Neuron, vol. 106, no. 5, 2020, https://doi.org/10.1016/j.neuron.2020.03.011. 

Li, Kun, et al. “ΒCaMKII in Lateral Habenula Mediates Core Symptoms of Depression.” Science, vol. 341, no. 6149, 2013, pp. 1016–1020., https://doi.org/10.1126/science.1240729.

Macosko, Evan&nbsp;Z., et al. “Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.” Cell, vol. 161, no. 5, 2015, pp. 1202–1214., https://doi.org/10.1016/j.cell.2015.05.002. 

McLaughlin, Ian, et al. “The Medial Habenula and Interpeduncular Nucleus Circuitry Is Critical in Addiction, Anxiety, and Mood Regulation.” Journal of Neurochemistry, vol. 142, 2017, pp. 130–143., https://doi.org/10.1111/jnc.14008. 

Mistry, Chetna J et al. “Genetics of Opioid Dependence: A Review of the Genetic Contribution to Opioid Dependence.” Current psychiatry reviews vol. 10,2 (2014): 156-167. doi:10.2174/1573400510666140320000928

Pandey, Shristi, et al. “Comprehensive Identification and Spatial Mapping of Habenular Neuronal Types Using Single-Cell RNA-Seq.” Current Biology, vol. 28, no. 7, 2018, https://doi.org/10.1016/j.cub.2018.02.040.

Sivalingam, Mageswary, et al. “Mapping of Morphine-Induced OPRM1 Gene Expression Pattern in the Adult Zebrafish Brain.” Frontiers in Neuroanatomy, vol. 14, 2020, https://doi.org/10.3389/fnana.2020.00005. 

Velasquez, Kenia M., et al. “The Role of the Habenula in Drug Addiction.” Frontiers in Human Neuroscience, vol. 8, 2014, https://doi.org/10.3389/fnhum.2014.00174. 

Wallace, Michael L, et al. “Anatomical and Single-Cell Transcriptional Profiling of the Murine Habenular Complex.” ELife, vol. 9, 2020, https://doi.org/10.7554/elife.51271. 


## Dataset identification
I am analyzing single-cell RNA sequencing data. 
The columns include: 

- *Barcode identity:* the barcode associated with a specific cell and the RNA expressed in the cell
- *orig.ident:* location/sample that the tissue was collected from. All of the rows have Hb because the target region to sequence was the habenula
- *nCount_RNA:* the number of RNA sequenced for the cell
- *nFeature_RNA:* the number of highly variable features (i.e, genes that are highly expressed in some cells, and lowly expressed in others)
- *percent.mt:* percent of the RNA that is mitochondrial


After running non-linear dimensional reduction on the data, I have another plot with:
- *p_val:* p_val (unadjusted)
- avg_logFC : log fold-change of the average expression between the two groups. Positive values indicate that the feature is more highly expressed in the first group. 
- *pct.1:* The percentage of cells where the feature is detected in the first group 
- *pct.2:* The percentage of cells where the feature is detected in the second group 
- *p_val_adj:* Adjusted p-value, based on Bonferroni correction using all features in the dataset. 
- *cluster number:* the label for the cluster is a single number 
- *gene:* gene that is associated with the cluster

## Workflow plan


- Set up libraries. Make sure they are installed.
- Load in the dataset
- *Quality control* and selecting cells for further analysis. Seurat allows you to easily explore QC metrics and filter cells based on any user-defined criteria
- We set of all genes starting with MT- as a set of mitochondrial genes. This is because non-viable cells have a high expression of mitochondrial RNA
- *Specify what cells we want to keep...* Here, we filter out cells that have unique feature counts over 2,500 or less than 200. We filter out cells that have >2% mitochondrial counts.
- *Normalize the data:* LogNormalize normalizes the feature expression measurements for each cell by the total expression
- Identification of highly variable features
- *Scaling the data:* Apply a linear transformation prior to dimensional reduction techniques like PCA. Shifts the expression of each gene, so that the mean expression across cells is 0 variance across cells is 1
- *Perform linear dimensional reduction:* Perform Principal Component Analysis (PCA) on the scaled data
- *Determine the ‘dimensionality’ of the dataset:* Identify ‘significant’ principal components (PCs) as those who have a strong enrichment of low p-value features
- *Cluster the Cells:* To cluster the cells, we next apply modularity optimization techniques to iteratively group cells together
- *Run non-linear dimensional reduction (UMAP/tSNE):* The goal of these algorithms is to place similar brain cells together into clusters
- *Finding differentially expressed features (cluster biomarkers):* Find markers for every cluster compared to all remaining cells, report only the positive ones
- *Assigning cell type identity to clusters:* use the known markers from past papers and label those clusters with the approprate brain cell-type name
- Subset the habenula neuron clusters from the brain cells, recluster to get subneuron clusters. Isolate the best gene markers for each cluster
- Compare the gene expression of the habenula 
- *Identify and visualize highly variable features:* The function FindVariableFeatures() and VariableFeaturePlot() directly models the mean-variance relationship inherent in single-cell data
- *VlnPlot() function shows expression probability distributions across clusters):* shows expression probability distributions across clusters
- *FeaturePlot() visualizes feature expression on a tSNE or PCA plot:* plot the location of specific genes and visualize where to find them and in which clusters they are highly expressed in
